lmdeploy/serve/async_engine.py

```diff
@@ -333,7 +333,7 @@ async def generate(
             self.id2step[str(session_id)] = step
         if gen_config is None:
             gen_config = GenerationConfig()
-        if isinstance(gen_config, GenerationConfig):
```

Comment:
What's the difference?

Reply:
if `gen_config` is type of `EngineGenerationConfig`, `isinstance(gen_config, GenerationConfig)` is true.